scss/page/page-form/page-form-content/_page-form-content-submit.scss

```diff
@@ -46,19 +48,30 @@
       background-color: $color-green-leaf-darker;
     }
     
-    &--recover-form-button {
+    &--recover-form-button,
+    &--subscribe-form-button {
       font-size: 18px;
 
       &-disabled {
         font-size: 18px;
         background-color: $color-mantis;
         border-color: $color-mantis;
         cursor: unset;
-
+        
         &:hover {
           background-color: $color-mantis;
         }
       }
-     }
+    }
+
+    &--subscribe-form-button-thanks {
```

Comment:
Nazwałbym to inaczej, bo jak ktoś sobie wpisze po prostu "No" w buttonie, to ta nazwa będzie kompletnie bez sensu, dlatego zawsze staraj się nazywać klasy, nie po tym co jest w kontencie, tylko jak np wygląda sekcja, co ma w sobie ect
tu można dać klase np button--red (bo jest jedyny taki, a to pewnie były button, do odmawiania) także red starczy

Reply:
Done